domains/adityagupta.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "RockstarBuddies",
+       "email": "yourrockingbuddy@gmail.com",
+       "discord": "899663426970865674"
+    },
+
+    "record": {
+        "CNAME": "2405:201:4003:4a4b::c0a8:1d01"
```

Comment:
We do not support AAAA records. Also, that is an invalid CNAME record.

Reply:
is this file even needed? I filed for "rockstarbuddies" one